src/pages/office/cat-cpc/office.cat-cpc.page.html

```diff
@@ -33,36 +43,59 @@ <h4>Test summary</h4>
 
           <div no-padding>
 
-            <candidate-description [display]="pageState.displayCandidateDescription$ | async" [formGroup]="form"
-                                   [candidateDescription]="pageState.candidateDescription$ | async"
-                                   [outcome]="pageState.testOutcome$ | async"
-                                   (candidateDescriptionChange)="candidateDescriptionChanged($event)">
+            <language-preferences
+              *ngIf="isDelegated"
+              [isDelegated]="isDelegated"
+              [formGroup]="form"
+              [isWelsh]="isWelsh()"
+              (welshChanged)="isWelshChanged($event)"></language-preferences>
+
+            <debrief-witnessed
+              [display]="isDelegated"
+              [isDelegated]="isDelegated"
+              [formGroup]="form"
+              [debriefWitnessed]="pageState.debriefWitnessed$ | async"
+              [outcome]="pageState.testOutcome$ | async"
+              (debriefWitnessedChange)="debriefWitnessedChanged($event)">
+            </debrief-witnessed>
+
+            <candidate-description
+              *ngIf="!(isDelegated && !isPass())"
```

Comment:
is !isPass here correct?

Reply:
you're right, at this stage it should only be isPass, if its not meant to be on the pass journey that will have to be amended again when we do the pass journey story, pushing a change now

Reply:
looks like candidate description is there for all 3 non delEx outcomes, and not at all for delEx.  Im only going off the miro screens tho.